maventa/MaventaService.system.test.ts

```diff
@@ -0,0 +1,32 @@
+import { MaventaService } from './MaventaService';
+import { MaventaConfig } from './MaventaConfig';
+import { HgNode } from '../../node/HgNode';
+
+describe('MaventaService System Tests', () => {
```

Comment:
Our other code uses style like:

```
describe('system', () => {
  describe('MaventaService', () => {
    describe('#method', () => {
      it('should ...', () => {
      });
    });
  });
});
``` 

This way it will be `system > MaventaService > #method` and all system tests can be browsed or disabled just by selecting the suite `system`, etc.